src/app.ts

```diff
@@ -24,6 +26,15 @@ import {
 } from './utils/config';
 
 export default async function (instance: FastifyInstance): Promise<void> {
+  await instance.register(swaggerPlugin, { openapi: { openapi: '3.0.0' } });
+
+  await instance.register(swaggerUiPlugin, {
```

Comment:
I this something we want to deploy in production ? I guess it is ok and this would provide some open documentation which is nice.

Reply:
I think so too !